Vagrantfile

```diff
@@ -313,9 +313,9 @@ Vagrant.configure(2) do |config|
 
```

Comment:
This commit looks like it fixes a (cosmetic or more serious?) issue primarily for the master branch. Can you cherry-pick it towards master?